src/countryInfo.js

```diff
@@ -0,0 +1,68 @@
+import { getAllApiData } from "./app";
+
+dataFilter()
+async function dataFilter() {
+
+    //filters data
+
+    let data = await getAllApiData()
+    let list = []
+
+    for (let item of data) {
+
+        list.push({
+            name: item.name,
+            flag: item.flag,
+            population: item.population,
+            color: regionColor(item.region)
+        })
+    }
+
+    list.sort((a, b) => { return a.population - b.population})
```

Comment:
Dit zou je nog korter kunnen schrijven
```suggestion
    list.sort((a, b) => a.population - b.population)
```